libraries/wallet/wallet.cpp

```diff
@@ -1162,6 +1175,20 @@ class wallet_api_impl
 
       return _builder_transactions[transaction_handle] = sign_transaction(_builder_transactions[transaction_handle], broadcast);
    }
+
+   pair<transaction_id_type,signed_transaction> broadcast_transaction(signed_transaction tx)
```

Comment:
@srpatel19590  - Please provide details of this code block addition.

Reply:
It is being called from `cli_multisig_transaction` cli test

```
// case2: broadcast without signature
      // expect: exception with missing active authority
      BOOST_CHECK_THROW(con.wallet_api_ptr->broadcast_transaction(transfer_tx2), fc::exception);
```